packages/compiler/lib/lib.tsp

```diff
@@ -124,18 +124,18 @@ scalar boolean;
  * Represent a model
  */
 // Deprecated June 2023 sprint
-@deprecated("object is deprecated. Please use {} for an empty model, `Record<unknown>` for a record with unknown propertie types, `unknown[]` for an array.")
+@deprecated("object is deprecated. Please use {} for an empty model, `Record<unknown>` for a record with unknown property types, `unknown[]` for an array.")
 model object {}
 
 /**
- * Array model type, equivalent to `T[]`
+ * @dev Array model type, equivalent to `T[]`
```

Comment:
What does the @dev tag signify?

Reply:
It is a current workaround so this doc doesn't carry over  to `model foo is x[]`

I filed an issue so we can discuss this pattern https://github.com/microsoft/typespec/issues/2003